.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+package-lock.json
```

Comment:
package-lock.json powinien trafić do repo
https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

Reply:
Nie tworzyłem go, ale z racji że nie jest potrzebny to dorzuciłem do ignora na poziomie wyższym niż frontend. Usunę na teraz, może nie będzie sie pojawiał. Inna sparawa że jak już rodzielimy na back i front do osobnych repo to nie będzie problemu.